src/Form/AssessorRequestType.php

```diff
@@ -50,14 +49,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         'widget' => PhoneNumberType::WIDGET_COUNTRY_CHOICE,
                     ])
                     ->add('emailAddress', EmailType::class)
-                    ->add('birthdate', BirthdayType::class, [
-                        'widget' => 'choice',
-                        'years' => $options['years'],
-                        'placeholder' => [
-                            'year' => 'AAAA',
-                            'month' => 'MM',
-                            'day' => 'JJ',
-                        ],
+                    ->add('birthdate', DateTimePickerType::class, [
+                        'max_date' => new \DateTime('-18 years'),
```

Comment:
15 vs 18 after this modif, it's ok?

Reply:
Here yes, 18 corresponds to the validation.